tenseal/binding.cpp

```diff
@@ -80,12 +81,16 @@ PYBIND11_MODULE(_tenseal_cpp, m) {
                                                       py::module_local())
         .def(py::init([](const shared_ptr<TenSEALContext> &ctx,
                          const vector<int64_t> &data) {
-            return BFVVector::Create(ctx, data);
-        }))
-        .def(py::init(
-            [](const shared_ptr<TenSEALContext> &ctx, const std::string &data) {
-                return BFVVector::Create(ctx, data);
-            }))
+                 return BFVVector::Create(ctx, data);
+             }),
+             py::call_guard<py::scoped_ostream_redirect,
+                            py::scoped_estream_redirect>())
```

Comment:
By default, pybind11 redirects all the logging done in c++ to /dev/null.
You must add this call_guard to redirect the logging to sys.stdout and sys.stderr.

Otherwise, it won't show the warnings for big vectors.